samples/ck-rx65n/basic-sample/src/demo_scanf.c

```diff
@@ -0,0 +1,53 @@
+#include <tx_api.h>
+
+#include "r_cg_macrodriver.h"
+#include <platform.h>
+#include <Config_SCI5.h>
+
+#include "demo_scanf.h"
+#include "demo_printf.h"
+#include "app_config.h"
+
+//
+// scanf() implementation only required for IOTCONNECT_INTERACTIVE_SYMMETRIC_KEY
+//
+#ifdef IOTCONNECT_INTERACTIVE_SYMMETRIC_KEY
+TX_MUTEX demo_scanf_mutex;
+TX_SEMAPHORE demo_scanf_semaphore;
+
+void demo_scanf_init(void)
+{
+    tx_mutex_create(&demo_scanf_mutex, "Demo scanf mutex", TX_INHERIT);
+
+    tx_semaphore_create(&demo_scanf_semaphore, "Demo scanf semaphore", 0u);
+}
+
+void demo_scanf_transmit_end(void)
+{
+    tx_semaphore_put(&demo_scanf_semaphore);
+}
+
+char my_sw_charget_function(void)
+{
+	char c = 0;
```

Comment:
Looks like some inconsistencies with indentation. Not a bit deal though.

Reply:
Stupid IDEs hiding TABs. Done.

Reply:
Happens to me all the time.